CONTRIBUTING.md

```diff
@@ -44,30 +44,43 @@ are more sensitive, emailed to <opensource@google.com>.
 
   1. Go makes it very simple to ensure properly formatted code, so always run
      `go fmt` on your code before committing it. You should also run
-     [golint][] over your code. As noted in the [golint readme][], it's not
-     strictly necessary that your code be completely "lint-free", but this will
-     help you find common style issues.
+     [go vet][] over your code. this will help you find common style issues
+     within your code and will keep styling consistent within the project.
 
-  1. Any significant changes should almost always be accompanied by tests. The
+  2. Any significant changes should almost always be accompanied by tests. The
```

Comment:
Could you please revert the changes to the numbering? I know it makes sense to have the correct number in the code, but with markdown it’s often just fine to designate that each of these is a list item, which can be done with `1.` [This article](https://www.seancdavis.com/posts/pro-tip-use-1-for-every-ordered-list-item-when-writing-markdown) talks about it.